site/content/user-guide/amqp-adapter.md

```diff
@@ -0,0 +1,175 @@
+* Message Body:
+  * (optional) Arbitrary payload
+* Message properties:
+  * (optional) Arbitrary properties (content-type, correlation-id, ...)
+* Disposition Frames:
+  * Accepted: Message successfully processed by the adapter.
+  * Released: Message cannot be processed and should be redelivered.
+  * Rejected: Adapter rejects the message due to:
+        * (`hono:bad-request`): A bad client request (e.g invalid content-type).
+        * (`amqp:unauthorized-access`): The adapter is disabled for tenant.
+        * (`amqp:precondition-failed`): Request not fulfilling certain requirements.
+
+This is the preferred way for devices to publish events. It is available only if the protocol adapter has been configured to require devices to authenticate (which is the default).
+
+**Example**
+
+Upload a JSON string for device `4711`:
+
+    $ qpid-send -b "localhost:4040" --connection-options "{username:sensor1@DEFAULT_TENANT, password:hono-secret, protocol:amqp1.0}" -a "'event'" --content-string '{"alarm": 1}'
+
+## Publish an Event (unauthenticated Device)
+
+* Address: `event/${tenant-id}/${device-id}` or `e/${tenant-id}/${device-id}`
+* Delivery: `AT_LEAST_ONCE`
+* Authentication: none
+* Message Header(s):
+  * (optional) Arbitrary headers (durable, ttl, priority, ...)
+* Message Body:
+  * (optional) Arbitrary payload
+* Message properties:
+  * (optional) Arbitrary properties (content-type, correlation-id, ...)
+* Disposition Frames:
+  * Accepted: Message successfully processed by the adapter.
+  * Released: Message cannot be processed and should be redelivered.
+  * Rejected: Adapter rejects the message due to:
+        * (`hono:bad-request`): A bad client request (e.g invalid content-type).
+        * (`amqp:unauthorized-access`): The adapter is disabled for tenant.
+        * (`amqp:precondition-failed`): Request not fulfilling certain requirements.
+
+This address format is used by devices that have not authenticated to the protocol adapter. Note that this requires the
+`HONO_AMQP_AUTHENTICATION_REQUIRED` configuration property to be explicitly set to `false`.
+
+**Examples**
+
+Publish some JSON data for device `4711`:
+
+    $ qpid-send -b "localhost:4040" --connection-options "{protocol:amqp1.0}" -a "'event/DEFAULT_TENANT/4711'" --content-string '{"alarm": 1}'
+
+## Downstream Meta Data
+
+Like the MQTT and HTTP adapters, the AMQP adapter also includes the following meta data in messages being sent downstream:
+
+| Name               | Location        | Type      | Description                                                     |
+| :----------------- | :-------------- | :-------- | :-------------------------------------------------------------- |
+| *orig_adapter*     | *application*   | *string*  | Contains the adapter's *type name* which can be used by downstream consumers to determine the protocol adapter that the message has been received over. The AMQP adapter's type name is `hono-amqp. |
```

Comment:
do you actually include these?

Reply:
yes, i think the generic `AbstractProtocolAdapterBase#newMessage(...)` implementation adds these for all protocol adapters (by default)